### ISSUE TITLE
Missing "./lib" export in "headers-polyfill" package

### ISSUE THREAD

**Issue by kingyue737** [mswjs/headers-polyfill#40]
I'm using `vite` with `mswjs`. After I bump versions of non-major dependencies, the following error is thrown caused by `headers-polyfill`. Currently, downgrading to v3.0.10 can fix it.

```
vite v3.1.1 building for production...
transforming...
✓ 25 modules transformed.
[commonjs--resolver] Missing "./lib" export in "headers-polyfill" package
file: /home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/msw@0.47.2_typescript@4.8.3/node_modules/msw/lib/index.js
error during build:
Error: Missing "./lib" export in "headers-polyfill" package
    at bail (file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/vite@3.1.1_sass@1.32.13+terser@5.15.0/node_modules/vite/dist/node/chunks/dep-a713b95d.js:33701:8)
    at resolve (file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/vite@3.1.1_sass@1.32.13+terser@5.15.0/node_modules/vite/dist/node/chunks/dep-a713b95d.js:33778:10)
    at resolveExports (file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/vite@3.1.1_sass@1.32.13+terser@5.15.0/node_modules/vite/dist/node/chunks/dep-a713b95d.js:34835:12)
    at resolveDeepImport (file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/vite@3.1.1_sass@1.32.13+terser@5.15.0/node_modules/vite/dist/node/chunks/dep-a713b95d.js:34853:31)
    at tryNodeResolve (file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/vite@3.1.1_sass@1.32.13+terser@5.15.0/node_modules/vite/dist/node/chunks/dep-a713b95d.js:3[45](https://github.com/kingyue737/vitify-admin/actions/runs/3059919545/jobs/4937844820#step:8:46)70:20)
    at Object.resolveId (file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/vite@3.1.1_sass@1.32.13+terser@5.15.0/node_modules/vite/dist/node/chunks/dep-a713b95d.js:34343:28)
    at file:///home/runner/work/vitify-admin/vitify-admin/node_modules/.pnpm/rollup@2.78.1/node_modules/rollup/dist/es/shared/rollup.js:22710:40
 ELIFECYCLE  Command failed with exit code 1.
Error: The process '/home/runner/setup-pnpm/node_modules/.bin/pnpm' failed with exit code 1
```

**Comment by kettanaito** [comment]
Hey. This has been fixed in `3.1`. Please update.

**Comment by kettanaito** [comment]
The update on the MSW side should follow later today.
